docs/guides/VTEX-Platform-Overview/Store-architecture/understanding-vtex-reference-architectures.md

```diff
@@ -0,0 +1,91 @@
+
+- [Macro elements: Layers](#macro-elements-layers)
+- [Micro elements: Components](#micro-elements-components)
+  - [Merchant Channels](#merchant-channels)
+  - [VTEX Core Services](#vtex-core-services)
+  - [Third-Party](#third-party)
+  - [Back Office](#back-office)
+
+## Captions, colors, and connections
+
+Different colors are used to identify the characteristics of the modules presented, such as where they are executed and which involved part provides them.
+
+Understand the meaning of each representation below:
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/Store-architecture/legend-and-colors.jpg)
+
+1. **Module code developed exclusively by VTEX with execution in VTEX infrastructure**: Pink background and pink border.
+2. **Module code not developed exclusively by VTEX (customizations) with execution in VTEX infrastructure**: White background and pink border.
+3. **Module code not developed exclusively by VTEX (customizations) with execution outside VTEX infrastructure**: White background and black border.
+4. **Back office services, which are executed outside VTEX infrastructure**: Gray background and black border.
+5. **Optional module (e.g., Integration Layers)**: Dashed border.
+
+The connection between modules and systems should be represented with arrows, which the arrowheads indicate the destination of the information and its return:
+
+6. **Black arrows**: Used for asynchronous calls, which are not real-time.
+7. **Blue arrows**: Used for synchronous calls, in real-time.
+
+## Macro elements: Layers
+
+The macro elements represent the entities involved in the project. In general, four major layers are considered: **Merchant Channels**, **VTEX Core Services**, **Third-Party**, and **Back Office**.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/Store-architecture/macro-elements-layers.jpg)
+
+Within each of these blocks, there are micro elements, which are the components that constitute the project. These components can be modules, apps, integration systems, sellers, sales channels, or any other component relevant to the architecture.
+
+## Micro elements: Components
+
+Now equipped with knowledge about the macro elements, it is possible to understand the components that constitute the four major layers: [Merchant Channels](#merchant-channels), [VTEX Core Services](#vtex-core-services), [Third-Party](#third-party), and [Back Office](#back-office).
+
+### Merchant Channels
+
+The merchant channels are the different digital sales channels used by the store, such as the website, PWA, and mobile app.
+
+From the architecture standpoint of ecommerce, the multiple channels should be highlighted to ensure that all stakeholders understand how the store interfaces with the end customer.
```

Comment:
Not sure I understand this comment. Looks more like internal guidelines to build a store architecture document rather than something focused on the audience.